cmd/kafka/main.go

```diff
@@ -20,9 +20,9 @@ import (
 )
 
 func main() {
-	sdk.Serve(
-		kafka.Specification,
-		kafka.NewSource,
-		kafka.NewDestination,
-	)
+	sdk.Serve(sdk.Connector{
+		NewSpecification: kafka.Specification,
+		NewSource:        kafka.NewSource,
+		NewDestination:   kafka.NewDestination,
+	})
```

Comment:
Can we please create a global object for `sdk.Connector`? We need it here, in the acceptance test and we will reference it when importing it as a built-in plugin in Conduit. I was thinking of adopting the same pattern in all connectors - a `connector.go` file in the root of the repo with a global variable called `Connector` and possibly a package doc:
```
// prefect place for a package doc...
package kafka

var Connector = sdk.Connector{
	NewSpecification: Specification,
	NewSource:        NewSource,
	NewDestination:   NewDestination,
}
```

Then we can reference it in Conduit as `kafka.Connector`.

Reply:
Sure, that sounds good to me.:+1: